packages/documentation/src/theme/Admonition/Types.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { ReactElement, ReactNode } from 'react';
+import DefaultAdmonitionTypes from '@theme-original/Admonition/Types';
+import styles from './styles.module.css';
+import { IxIcon } from '@siemens/ix-react';
+
+interface LayoutProps {
+  readonly children: ReactNode;
+}
+
+function Layout({ children }: LayoutProps): JSX.Element {
+  return <div className={`${styles.layout}`}>{children}</div>;
+}
+
+const ICON_CONFIG: Record<string, { name: string; color: string }> = {
+  do: { name: 'success', color: 'color-success' },
+  dont: { name: 'cancelled', color: 'color-alarm' },
+  doGradient: { name: 'success', color: 'color-success' },
+  dontGradient: { name: 'cancelled', color: 'color-alarm' },
+  default: { name: null, color: null },
```

Comment:
Use esm import from `@siemens/ix-icons/icons` e.g `import { iconSuccess } from '@siemens/ix-icons/icons'` and not the string variant.